site/pt/beta/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>I would translate with a few differences, to match the original text:</p><blockquote>As avaliações—<strong>os</strong>&nbsp;<em>arrays</em>&nbsp;de inteiros— <strong>devem</strong> ser <strong>convertidas</strong> em tensores (<em>tensors</em>) antes de alimentar a rede neural.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>